mne/time_frequency/tfr.py

```diff
@@ -245,6 +247,253 @@ def _cwt(X, Ws, mode="same", decim=1, use_fft=True):
         yield tfr
 
 
+# Loop of convolution: single trial
+
+
+def _compute_tfr(epoch_data, frequencies, sfreq=1.0, method='morlet',
+                 n_cycles=7.0, zero_mean=None, time_bandwidth=None,
+                 use_fft=True, decim=1, output='complex', n_jobs=1,
+                 verbose=None):
+    """Computes time-frequency transforms.
```

Comment:
I'd defer to the 'pedia, plus I think it's a compound adjective which would suggest [to hyphenate] (https://www.google.com/?ion=1&espv=2#q=compound%20adjective%20hyphenation)